pkg/cmd/cli/cmd/set/env.go

```diff
@@ -48,6 +48,12 @@ syntax.`
   # Update all containers in all replication controllers in the project to have ENV=prod
   %[1]s env rc --all ENV=prod
 
+  # Import environment from a secret
+  %[1]s env --from-secret=mysecret dc/myapp
```

Comment:
> Ha, @smarterclayton preferred it the way Seth did it (well, to be fair, the proposal was --from-resource and not --from).

I like `--from=resource/name` too.  Given a syntax that lets me pick a particular field and generic protections against escalations, it would be neat to be able to collect env from any object in the namespace.